lib/remote_resource/request.rb

```diff
@@ -80,6 +80,8 @@ def body
       @body ||= begin
         if [:put, :patch, :post].include?(http_action)
           attributes.to_json
+        elsif http_action == :get && connection_options[:force_get_params_in_body]
+          connection_options[:params].to_json
         else
           nil
         end
```

Comment:
A case statement would be bit more readable here I think:
```suggestion
        case http_action
        when :put, :patch, :post
          attributes.to_json
        when :get
          connection_options[:params].to_json if connection_options[:force_get_params_in_body]
        else
          nil
        end
```